.github/workflows/handle.yml

```diff
@@ -50,9 +50,17 @@ jobs:
     permissions:
       pull-requests: 'read'
     outputs:
-      REVIEW_DECISION: '${{ steps.get_review_decision.outputs.REVIEW_DECISION }}'
+      REVIEW_DECISION: '${{ steps.get_breakglass_decision.outputs.REVIEW_DECISION || steps.get_review_decision.outputs.REVIEW_DECISION }}'
     steps:
+      - id: 'get_breakglass_decision'
+        if: |-
+          contains(github.event.pull_request.labels.*.name, 'aod-breakglass')
```

Comment:
Nit: should it be github.event.label.name == 'aod-breakglass' ? Since handle will now also be triggered by labeling event.

Thank you for making this change!

Reply:
I think this syntax will stil work and it will work across event types. There might be an issue here though because pull request and pull request review show up as different status checks....let me dig in after I merge this and test it on one of our repos.

Reply:
Sounds like a plan!